folium/features.py

```diff
@@ -256,13 +256,10 @@ def render(self, **kwargs):
             """).render(this=self, kwargs=kwargs)), name=self.get_name())
 
         self._parent.script.add_child(Element(Template("""
-            var embedSpec = {
-                mode: "vega-lite",
-                spec: {{this.json}}
-            };
-            vg.embed(
-                {{this.get_name()}}, embedSpec, function(error, result) {}
-            );
+            const spec = {{this.json}};
+            vegaEmbed({{this.get_name()}}, spec)
```

Comment:
It seems `vegaEmbed` is the function to use in vega embed version 3, while `vg.embed` was used in version 2, is that right?

Reply:
yes, that seems to be the case.